[docs] [#11472] Removed manual transaction table creation from geo-partioning example (#11474)
yugabyte/yugabyte-db: docs/content/latest/admin/yb-admin.md

---

> {{< note title="Notes" >}}
> 
> The preferred way of creating transaction status tables when using YSQL is to simply create a tablespace with the appropriate placement. A transaction table using the tablespace's placement will automatically be created when you create the first table using the tablespace.
> 
> {{< /note >}}

Reviewer: Does this apply to YCQL at all?

Also, is it even possible to create a transaction table manually anymore? If not, we can reword this note some more to say something like: "You don't need to manually create transaction status tables. Create a tablespace with the appropriate placement, and when you create the first table using that table space, YugabyteDB automatically creates a transaction table using the tablespace's placement."

(Or even, "Don't manually create transaction status tables. Create a ...")

```suggestion
{{< note title="Note" >}}

The preferred way to create transaction status tables with YSQL is to create a tablespace with the appropriate placement. YugabyteDB automatically creates a transaction table using the tablespace's placement when you create the first table using the new tablespace.

{{< /note >}}
```

Author: Manual table creation is in a bit of an interesting state at the moment. It's definitely still possible (yb-admin commands are not removed), but shouldn't be used for YSQL, though if created will be used (but also redundant, since the automatic ones will overlap). The commands remain for future support for YCQL, but the tables aren't used for YCQL at the moment and won't be until future changes are made (local to global promotions).
